docs/astro.config.mjs

```diff
@@ -179,7 +179,12 @@ export default defineConfig({
 					autogenerate: { directory: 'resources' },
 				},
 			],
-			expressiveCode: { shiki: { langs: [markdocGrammar] } },
+			expressiveCode: {
+				shiki: {
+					// TODO: Remove `any` once Expressive Code supports grammars with comments.
```

Comment:
For reference, should be fixed with [`f594927` (#2551)](https://github.com/withastro/starlight/pull/2551/commits/f594927614879a3e290c48a197e5aed7dabb3cd0)